.github/workflows/crowdin.yml

```diff
@@ -0,0 +1,31 @@
+name: Crowdin Action
+
+on:
+  push:
+    paths: [
+      'src/locale/**',
+      'src-ui/src/locale/**'
+    ]
+    branches: [ dev ]
+
+jobs:
+  synchronize-with-crowdin:
+    runs-on: ubuntu-latest
+
+    steps:
+
+    - name: Checkout
+      uses: actions/checkout@v4
+
+    - name: crowdin action
+      uses: crowdin/github-action@v1
+      with:
+        upload_translations: false
```

Comment:
In the migration guide, I see they have this set to true, but I'm also not clear what it does

Reply:
Yea its confusing, but I think we dont want this, we just download our translations from crowdin, this would allow things both ways (ie translation here changes we upload back to there).